Documentation/Upgrade/README.rst

```diff
@@ -0,0 +1,106 @@
+.. _README:
+
+.. include:: ../lib/doc-includes/pike-icons.inc
+
+.. Don't use default python highlighting for code blocks http://www.sphinx-doc.org/en/stable/markup/code.html
+
+.. highlight:: none
+
+
+===================================
+Minor Upgrade to Release 5.4.1 U8+
+===================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Use this documentation when upgrading between minor releases. The process applies to upgrades from Release 5.4.1 U5 to Release 5.4.1 U8+. During this process, the Nuage components and Red Hat can be updated at the same time.

```

Comment:
U5 or U5+  (Question to Andre)

Reply:
I will check with Andre and update when I get an answer.

Reply:
done